content/docs/reference/compatibility.md

```diff
@@ -16,29 +16,39 @@ Neon cloud service does not currently provide the user with access permissions o
 
 <a id="default-extensions/"></a>
 
-## Default Extensions
+## Available PostgreSQL Extensions
 
 During technical preview Neon has restrictions on user ability to install PostgreSQL extensions.
 
-Following PostgreSQL extensions come pre-installed:
-
-|               | Version | Note |
-| ------------- | ------- | ---- |
-| plpgsql       | 1.0     |      |
-| fuzzystrmatch | 1.1     |      |
-| tcn           | 1.0     |      |
-| unaccent      | 1.1     |      |
-| btree_gin     | 1.3     |      |
-| btree_gist    | 1.6     |      |
-| citext        | 1.6     |      |
-| cube          | 1.5     |      |
-| dict_int      | 1.0     |      |
-| hstore        | 1.8     |      |
-| intarray      | 1.5     |      |
-| isn           | 1.2     |      |
-| lo            | 1.1     |      |
-| ltree         | 1.2     |      |
-| seg           | 1.4     |      |
+The following PostgreSQL extensions can be installed by the user:
+
+|                         | Version | Note |
+|:------------------------|--------:|------|
+| btree_gin               |     1.3 |      |
+| btree_gist              |     1.6 |      |
+| citext                  |     1.6 |      |
+| cube                    |     1.5 |      |
+| dict_int                |     1.0 |      |
+| fuzzystrmatch           |     1.1 |      |
+| hstore                  |     1.8 |      |
+| intarray                |     1.5 |      |
+| isn                     |     1.2 |      |
+| lo                      |     1.1 |      |
+| ltree                   |     1.2 |      |
+| pg_trgm                 |     1.6 |      |
+| pgcrypto                |     1.3 |      |
+| plpgsql                 |     1.0 | Comes pre-installed with postgres |
+| postgis                 |   3.3.0 |      |
+| postgis_raster          |   3.3.0 |      |
+| postgis_tiger_geocoder  |   3.3.0 | Cannot be installed through the web UI - use your psql user credentials instead. |
```

Comment:
| postgis_tiger_geocoder  |   3.3.0 | Cannot be installed using the Neon web UI. Use your `psql` user credentials to install. |